src/uwtools/apps/srw.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+#pylint: disable=unused-import, unused-variable, unused-argument, useless-parent-delegation
+# remove these disables once implemented
+'''
+This file contains the specific drivers for a particular app, using the facade pattern base class.
+'''
+
+import logging
+import os
+import sys
+import shutil
+import subprocess
+
+from uwtools.utils import file_helpers
+from uwtools.utils import cli_helpers
+from ..drivers.facade import Facade
+
+logging.getLogger(__name__)
+
+
+class SRW210(Facade): # pragma: no cover
+    #remove pragma when completed
+
+    '''
+        Concrete class to handle UFS Short Range Weather app forecasts.
+    '''
+    def __init__(self, argv):
```

Comment:
I believe it will be an important and useful refactor and new requirement. Since we have several other instances that will require significant changes, I'd prefer not to blow @WeirAE's work out of scope and include that requirement on this work. 

The only caveat being that if we don't get the work done on this PR in a timeline that the refactor beats it. I certainly hope that is not the case given we haven't actually planned that work yet!

Reply:
Totally agreed!